test/cli.js

```diff
@@ -20,50 +19,6 @@ function execCli(args, cb) {
 	}, cb);
 }
 
-test('display test title prefixes', function (t) {
-	t.plan(6);
-
-	execCli([
-		'fixture/async-await.js',
-		'fixture/es2015.js',
-		'fixture/generators.js'
-	], function (err, stdout, stderr) {
-		t.ifError(err);
-
-		// remove everything except test list
-		var output = stderr
-			.replace(/[0-9] tests passed/, '')
-			.replace(new RegExp(figures.tick, 'gm'), '')
-			.replace(/^\s+/gm, '')
-			.trim();
-
-		var separator = ' ' + figures.pointerSmall + ' ';
-
-		// expected output
-		var tests = [
-			['async-await', 'async function'].join(separator),
```

Comment:
The whole point of this test is that the file names get prefixed. We still need to test for this whole string.